### DISCUSSION TITLE
What's the recommended development lifecycle for Directus?

### DISCUSSION THREAD

**Discussion by MattElcock** [directus/directus#10581] [Archive]
Hey guys. 

I'm looking at Directus as a quick and easy way to spin up a backend for a proof of concept for a new product at work. Interested to see what people's experiences have been using Directus, what their workflow is like when developing with it, and any super cool features you think put it ahead of something like Strapi.

**Comment by rijkvanzanten** [comment]
I think I'm way too biased to answer these questions myself 😁 Happy to discuss if you have specific questions though!
